cmd/registry.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/deis/pkg/prettyprint"
+
+	"github.com/deis/workflow-cli/controller/api"
+	"github.com/deis/workflow-cli/controller/models/config"
+)
+
+// RegistryList lists an app's registry information.
+func RegistryList(appID string) error {
+	c, appID, err := load(appID)
+
+	if err != nil {
+		return err
+	}
+
+	config, err := config.List(c, appID)
+
+	fmt.Printf("=== %s Registry\n", appID)
+
+	registryMap := make(map[string]string)
+
+	for key, value := range config.Registry {
```

Comment:
Same reason as tags, I wanted to do a short cut tho - maybe we can unify them all later?